docs/general-reference/settings.md

```diff
@@ -1,123 +1,87 @@
 
+For more information, see our <a href="https://testobject.com/pricing" target="_blank">plans and pricing page</a>.
 
-<h4>Manual Testing Plans</h4>
-<img class="left shadow" src="/img/settings/manual-bar.png">
+<h3 id="pro_plans">Professional Plans</h3>
 
-You will have the choice between a <b>Single Plan</b> or a <b>Team Plan</b>.
+* <a href="#manual_plan">Manual Testing plans</a> (a Single Plan or a Team Plan).
+* <a href="#automated_plan">Automated Testing plans</a> (a Starter Plan or an Advanced Plan).
+* <a href="#enterprise_plan">Enterprise Plans</a>
 
-The plans differ in the number of users, parallel sessions and the available device time a day.
 
-For more information see the <a href="https://testobject.com/pricing" target="_blank">overview for all plans</a> on our homepage.
+<h4 id="manual_plan">Manual Testing Plans</h4>
+<img class="left shadow" src="/img/settings/manual-bar.png">
 
+You will have the choice between a <b>Single Plan</b> or a <b>Team Plan</b>.<br/>
+The plans differ in the number of users, parallel sessions and the available device time a day.<br/>
+For more information, see our <a href="https://testobject.com/pricing" target="_blank">plans and pricing page</a>.
 
-<h4>Automated Testing plans</h4>
+<h4 id="automated_plan">Automated Testing plans</h4>
 <img class="left shadow" src="/img/settings/automated-bar.png">
 
-For automated testing you can purchase the <b>Starter Plan</b> or the <b>Advanced plan</b>.
+For automated testing you can purchase the <b>Starter Plan</b> or the <b>Advanced plan</b>.<br/>
+Both plans are accessible for all users of your <a href="#team-accounts">team</a>. They differ in the number of possible concurrent test runs, the available device time and the priority of the test execution.<br/>
+For more information, see our <a href="https://testobject.com/pricing" target="_blank">plans and pricing page</a>.
 
-Both plans are accessible for all users of your <a href="#team-accounts">team</a>. They differ in the number of possible concurrent test runs, the available device time and the priority of the test execution.
+<h4 id="enterprise_plan">Enterprise Plans</h4>
+Additionally to our standard plans we offer flexible, individual solutions for our customers.
 
-For more information see the <a href="https://testobject.com/pricing" target="_blank">overview for all plans</a> on our homepage.
+These plans are individually customizable. If you are not sure which plan to use, please contact our Sales Team: <a href="mailto:sales@testobject.com">sales@testobject.com</a> to discuss your needs.
 
-<h4>Enterprise Plans</h4>
-Additionally to our standard plans we offer flexible, individual solutions for our customers. 
+<h3 id="billing">Billing</h3>
+Here you see your payment data and your billing history. Update your credit card information and check all your previous payments made to TestObject.
```

Comment:
Second phrase is disconnected from previous one + is using imperative ("Update your credit card information!!"). Rephrase.